images/federation-v2/Dockerfile

```diff
@@ -13,9 +13,10 @@
 # limitations under the License.
 
 
-FROM ubuntu:14.04
+FROM ubuntu:latest
```

Comment:
why use `latest`? I think we should use a stable version, such as `18.03` etc?

Reply:
The latest is a stable version. That is, the ubuntu:latest tag points to the "latest LTS", since that's the version recommended for general use.